docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -64,26 +64,29 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 #### Q: should the number of retries of a job be limited, and if so, where? e.g. a max_retries field in the parent job? wait and see whether people attempt to rerun jobs that have already failed nine zillion times?
 A: Unknown TBD
 
-#### Q: Preventing the same params from being re-run
-A: We have decided to allow it
+#### Q: How do we prevent jobs with identical parameters from being rerun more than once within a retry_jobs request?
+A: We have decided to allow multiple jobs with the same params to be re-run in the same `retry_jobs` request.
 
-#### Q: Finding the most recent run of the job: I would very much like to avoid anything involving iterating over a chain of jobs before you can find the most recent run or the original run -- we can come up with better data structures than that!
-A: Unknown TBD, maybe the frontend does it?
+#### Q: How do we find the most recent retry of a job?
+A: The client using the ee2 API would have to figure it out using the `retry_parent` and job creation date fields. Unless we added 
 
 #### Q: It might be best to always submit a git commit for the module, maybe?
```

Comment:
Reword to:

Q. How do we ensure that the correct version of an app is always run?

A. The git commit for an app is the most accurate way to specify the version. Ideally any clients using the ee2 API would supply this information.

Reply:
The clients mostly use "dev/beta/release" within the app runs, but this was actually meant to talk about how the narrative sends dev/beta/release instead of git commits, so fixed it up to reflect it.